zkevm-circuits/src/witness/tx.rs

```diff
@@ -161,6 +233,7 @@ pub(super) fn tx_convert(
             .input
             .iter()
             .fold(0, |acc, byte| acc + if *byte == 0 { 4 } else { 16 }),
+        chain_id: 1, // FIXME
```

Comment:
is chain_id available in circuit_input_builder::Transaction?

Reply:
No. It is included in `circuit_input_builder::BlockHead` (I've made the changes).